src/scripts/main.js

```diff
@@ -1,5 +1,29 @@
 'use strict';
 
+const wall = document.querySelector('.wall');
+const wallHeight = wall.offsetHeight;
+const wallWidth = wall.offsetWidth;
+
+const spider = document.querySelector('.spider');
+const spiderWidth = spider.offsetWidth;
+const spiderHeight = spider.offsetHeight;
+
 document.addEventListener('click', e => {
-  // write code here
+  if (wall.contains(e.target)) {
```

Comment:
instead of nesting other code in this first if-statement, just check the condition (wall.contains(e.target)) and return nothing if it's not true

Reply:
ok, the code will be more readable. Thanks